# Define the goal
string fastening_tool_name
uint32 speed
float32 duration
string direction
---
# Define the result
bool control_result
---
# Define a feedback message
uint32 motor_speed